console/console/build_command.go

```diff
@@ -0,0 +1,103 @@
+package console
+
+import (
+	"fmt"
+	"golang.org/x/exp/slices"
```

Comment:
The `exp` package is unstable, we should not use it, please use https://pkg.go.dev/slices instead.